package.json

```diff
@@ -16,12 +16,22 @@
     "version-packages": "changeset version",
     "release": "turbo run build --filter=login^... && changeset publish"
   },
+  "pnpm": {
+    "overrides": {
+      "@typescript-eslint/parser": "^7.9.0"
+    }
+  },
   "devDependencies": {
     "@changesets/cli": "^2.22.0",
-    "eslint": "^7.32.0",
+    "@vitejs/plugin-react": "^4.2.1",
+    "eslint": "^8.57.0",
     "eslint-config-zitadel": "workspace:*",
     "prettier": "^3.2.5",
-    "turbo": "^1.10.8"
+    "tsup": "^8.0.2",
+    "turbo": "^1.10.8",
+    "typescript": "^5.4.5",
+    "vite-tsconfig-paths": "^4.3.2",
+    "vitest": "^1.6.0"
```

Comment:
Moved some dev tools dependencies to the root for simple management across the mono repo.